shopgoodwill.py

```diff
@@ -415,6 +419,39 @@ def get_query_results(
                 ):
                     return total_listings
 
+    def get_item_shipping_estimate(self, item_id: int, zip_code: str) -> float:
```

Comment:
nitpick - the return type should be Optional[float], due to line 452

Reply:
Fixed in e14a84f